MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -586,11 +586,25 @@ public final class MagazineLayout: UICollectionViewLayout {
   }
 
   override public func shouldInvalidateLayout(forBoundsChange newBounds: CGRect) -> Bool {
-    let isSameWidth = collectionView?.bounds.size.width.isEqual(
-      to: newBounds.size.width,
-      threshold: 1 / scale)
-      ?? false
-    return !isSameWidth || hasPinnedHeaderOrFooter
+    // When using the topToBottom layout direction, we only want to invalidate the layout when the
+    // widths differ. When using the bottomToTop layout direction, we want to invalidate on any
+    // size change due to the requirement of  needing to preserve scroll position from the bottom
+    var shouldInvalidateDueToSize = false
```

Comment:
Are we able to use an uninitialized let here?

Reply:
Whoops, yeah we should have. @elfredpagan can you follow up with a fix for that?